csrc/index_compute.cpp

```diff
@@ -48,15 +48,19 @@ int getProducerHaloOffset(
                  .mapDifferentExtents(true)
                  .mapProducerToConsumer();
 
-  auto producer_id = producer_tv->getMaybeRFactorDomain()[producer_axis];
+  auto producer_id = producer_tv->getMaybeAllocationDomain()[producer_axis];
```

Comment:
This change is just so we have better consistency with our usage of `producer_axis`